app/editor/src/hooks/useTooltips.ts

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactTooltip from 'react-tooltip';
+
+/**
+ * Tooltips require being rebuilt when a component is created or they will not be displayed.
+ */
+export const useTooltips = () => {
```

Comment:
@jdtoombs We should start using this hook to keep things clean going forward.

Reply:
I like it